packages/node_modules/nav-frontend-tabs/src/tab.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
+import * as cn from 'classnames';
+import { omit } from 'nav-frontend-js-utils';
+
+const tabCls = (props) => cn('nav-frontend-tabs__tab-inner', {
+    'nav-frontend-tabs__tab-inner--aktiv': props.aktiv,
+    'nav-frontend-tabs__tab-inner--interaktiv': !props.aktiv,
+    'nav-frontend-tabs__tab-inner--disabled': props.disabled
+});
+
+export interface TabProps {
+    label?: React.ReactNode | React.ReactChild | React.ReactChildren;
+    aktiv: boolean;
+    onClick?: (event: React.MouseEvent<HTMLButtonElement>) => void;
+    disabled?: boolean;
+}
+
+class Tab extends React.Component<TabProps> {
+    getLabel() {
+        return (
+            <div className="nav-frontend-tabs__tab-label">
+                {this.props.label || this.props.children}
+            </div>
+        );
+    }
+
+    getInnerTab() {
+        if (!this.props.aktiv) {
+            return (
+                <button
```

Comment:
I eksisternde løsning som bruker en ligende komponent er hver enkelt tab en lenke.
Burde vi lagt til muligheten for å styre hva slags komponent som blir brukt?

Reply:
Hm, mulig. I utgangspunktet tenker jeg at knapper er fordelaktig i og med at de aktiveres både med space og enter som default. Men igjen; dette er sikkert noe David kan gi oss en anbefaling på.

Reply:
Nokså sikker på at om vi bare tillatter `button` så vil vi få problemer med andre UU-krav, f.eks da folk begynner å endre på urlen basert på hvilken tab som er valgt.